src/gluonts/dataset/parallelized_loader.py

```diff
@@ -438,8 +478,9 @@ class ParallelDataLoader(object):
         Floating point type to use.
     shuffle
         Whether to shuffle the samples.
-    sampler
-        The sampler to use. Either specify sampler or shuffle, not both.
+    num_batches_for_shuffling
+        The number of batches among which samples are shuffled. So for example if num_batches_for_shuffling = 8
+        then the next num_batches_for_shuffling * 8 samples will be shuffled and then batched.
```

Comment:
If `num_batches_for_shuffling = 8` then `num_batches_for_shuffling * 8 = 64`? Or did you mean `batch_size * 8`?

Reply:
I should maybe say: if num_batches_for_shuffling = 8 and batch_size=8 then effectively 64 samples will be shuffled and then batched.